test/verify/check-machines-virsh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
```

Comment:
This needs to be Python 3 now, and the `coding` is obsolete with py3.

Reply:
Done